### PR TITLE
Task8 patch

### PR DESCRIPTION
**Pull request by reemhd**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -213,7 +213,91 @@ describe("POST method", () => {
...
+describe("PATCH method", () => {
+  describe("/api/articles/:article_id", () => {
+    it("200 response with an increase of votes in article by id", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 2 })
+        .expect(200)
+        .then(({ body }) => {
+          const updatedArticle = body.updated
+          const orginalArticle = {
```
Comment (corkesw) [inline __tests__/app.test.js]: I don't see this being used so it's best to delete it

File: __tests__/app.test.js [hunk 2]
```diff
@@ -213,7 +213,91 @@ describe("POST method", () => {
...
+        });
+    });
+    it("200 response with an decrease of votes in article by id", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: -2 })
+        .expect(200)
+        .then(({ body }) => {
+          const updatedArticle = body.updated;
+          const orginalArticle = {
```
Comment (corkesw) [inline __tests__/app.test.js]: Same here!

File: __tests__/app.test.js [hunk 3]
```diff
@@ -213,7 +213,91 @@ describe("POST method", () => {
...
+          expect(body.message).toBe("Bad request");
+        });
+    });
+    it("400 status code if invalid object sent", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({})
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Invalid request");
```
Comment (corkesw) [inline __tests__/app.test.js:278]: test suite is very thorough, nice work 🙌

**Review by corkesw — COMMENTED** [review]
Great work and progress, just a little tidying in the test suite and then this look great to merge 🙂

File: __tests__/app.test.js (hunk 3 above)
Comment (reemhd) [reply __tests__/app.test.js:278]: thanks Stephen! cleaned up the test suite :)

**Pull request merged**